examples/server/server.cpp

```diff
@@ -2230,6 +2233,7 @@ struct server_context {
 
                     auto res = std::make_unique<server_task_result_metrics>();
                     res->id                  = task.id;
+                    res->slots_data          = slots_data;
```

Comment:
This should save some copies:

```suggestion
                    res->slots_data          = std::move(slots_data);
```